codespell_lib/data/dictionary.txt

```diff
@@ -15379,22 +15381,31 @@ severitie->severity, severities,
 severley->severely
 severly->severely
 sevice->service
+sewdonim->pseudonym
 sgadow->shadow
 sh1sum->sha1sum
 shadasloo->shadaloo
 shaddow->shadow
 shadhow->shadow
 shadoloo->shadaloo
 shamen->shaman, shamans,
+shandeleer->chandelier
 shandow->shadow
+shaneal->chenille
+shanghi->Shanghai
 sharable->shareable
+sharloton->charlatan
+sharraid->charade
+sharraids->charades
 shashes->slashes
+shatow->chateaux
```

Comment:
This is the plural, the singular is château (apparently the plural can be x or s).

Reply:
Done